src/commands/games/blackjack.ts

```diff
@@ -13,37 +13,24 @@ import {
 } from '../../components/games/blackjack';
 import { getEmojiByName } from '../../components/emojis';
 import { adjustCoinBalanceByUserId, getCoinBalanceByUserId, UserCoinEvent } from '../../components/coin';
+import { BOT_PREFIX } from '../../bot';
 
 const DEFAULT_BET = 10;
 const MIN_BET = 10;
 const MAX_BET = 1000000;
 
-const validateBetAmount = (amount: number): string | boolean => {
+const validateBetAmount = (amount: number): string => {
   if (amount < MIN_BET) return `minimum bet is ${MIN_BET} Codey coins.`;
   if (amount > MAX_BET) return `maximum bet is ${MAX_BET} Codey coins.`;
-  return true;
+  return '';
 };
 
-class BlackjackCommand extends BaseCommand {
-  constructor(client: CommandoClient) {
-    super(client, {
-      name: 'blackjack',
-      aliases: ['blj', 'bj'],
-      group: 'games',
-      memberName: 'blackjack',
-      description: 'Start a Blackjack game to win some Codey coins!',
-      examples: [`${client.commandPrefix}blackjack 100`, `${client.commandPrefix}blj 100`],
-      args: [
-        {
-          key: 'bet',
-          prompt: `enter the amount you want to bet.`,
-          type: 'integer',
-          default: DEFAULT_BET
-        }
-      ]
-    });
-  }
-
+@ApplyOptions<CommandOptions>({
+  aliases: ['blj', 'bj'],
+  description: 'Start a Blackjack game to win some Codey coins!',
+  detailedDescription: `**Examples:** \`${BOT_PREFIX}blackjack 100\`, \`${BOT_PREFIX}blj 100\``
+})
+export class BlackjackCommand extends Command {
```

Comment:
bruhh this is so cool
is there no longer a name/group/membername needed? 
how might this effect our .help command?

Reply:
Yeah, it looks like it is done automatically. https://www.sapphirejs.dev/docs/Documentation/api-framework/classes/Command#fullcategory
I don't think Sapphire comes with a built-in help command, unfortunately.